src/olympia/core/tests/test_babel.py

```diff
@@ -0,0 +1,26 @@
+from io import BytesIO
+
+import pytest
+from babel.messages.extract import DEFAULT_KEYWORDS
+
+from olympia.core.babel import extract_jinja, generate_option
+
+
+@pytest.mark.parametrize(
+    'value,expected',
+    [
+        (True, 'true'),
+        (False, 'false'),
+        ('foo', 'foo'),
+        (['foo', 'bar'], 'foo,bar'),
+        (('abc', 'def'), 'abc,def'),
+    ],
+)
+def test_generate_option(value, expected):
+    assert generate_option(value) == expected
+
+
+def test_extract_jinja_no_options():
+    # Doesn't actually extract, just tests that extract_jinja() doesn't fail
+    # when converting our settings in options for the underlying function.
+    extract_jinja(BytesIO(), DEFAULT_KEYWORDS, ['L10n:'], {})
```

Comment:
It's out of scope for this but I think we should consider run a true extract (without merging the locale files) as part of the test suite. This would ensure we can actually run the extraction at all times on `master` branch.

Reply:
Commented in https://github.com/mozilla/addons-server/issues/9112#issuecomment-1094975106